control/controlclient/direct.go

```diff
@@ -126,9 +126,9 @@ type Options struct {
 
 // Pinger is a subset of the wgengine.Engine interface, containing just the Ping method.
 type Pinger interface {
-	// Ping is a request to start a discovery or TSMP ping with the peer handling
-	// the given IP and then call cb with its ping latency & method.
-	Ping(ip netaddr.IP, useTSMP bool, cb func(*ipnstate.PingResult))
+	// Ping is a request to start a ping with the peer handling the given IP and
+	// then call cb with its ping latency & method.
+	Ping(ip netaddr.IP, pingType string, cb func(*ipnstate.PingResult))
```

Comment:
clarify the set of valid pingTypes for this string. (especially important as this is the interface's docs)

maybe we want a `type PingType string` somewhere onto which we can attach docs?

Reply:
I started doing that, but it became a cyclic import problem. I'll go back and do it and introduce a package if needs be.